src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,107 @@
+import { NavLink } from 'react-router-dom';
+import './Sidebar.scss';
+
+//*import React, { SetStateAction, useContext } from 'react';*//
+//*import classNames from 'classnames';*//
+//*import { Logo } from '';*//
+//*import { FavouritesContext } from '';*//
+//*import { CartContext } from '';*//
+
+
+
+
+type Props = {
+  isMenuOpen: boolean;
+  setIsMenuOpen: React.Dispatch<SetStateAction<boolean>>;
+};
+
+export const SideNavbar: React.FC<Props> = ({ isMenuOpen, setIsMenuOpen }) => {
+  //**const { favourites } = useContext(FavouritesContext);**//
+  //**const { cartList } = useContext(CartContext);**//
+  const productsInCart = cartList.reduce((acc, item) => acc + item.quantity, 0);
+
+  const getLinkClass = ({ isActive }: { isActive: boolean }) =>
+    classNames('navbar-link', {
+      'navbar-link-active': isActive,
+    });
```

Comment:
Good use of getLinkClass to dynamically apply classes based on the active state!